examples/v1/pipelineruns/no-ci/cluster-resolver.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: tekton.dev/v1
```

Comment:
This is copied from `v1beta1` examples, and I assume it is because that we have to create some tasks in the cluster ourselves to use this as for the `some-pipeline` in the `pipelineRef`.